NorthstarDLL/scripts/scriptjson.cpp

```diff
@@ -212,10 +212,12 @@ ADD_SQFUNC(
 			doc.GetErrorOffset());
 
 		if (bFatalParseErrors)
+		{
 			g_pSquirrel<context>->raiseerror(sqvm, sErrorString.c_str());
-		else
-			spdlog::warn(sErrorString);
+			return SQRESULT_ERROR;
```

Comment:
```suggestion
```

Reply:
Not adding this because it isn't consistent with the rest of the repository.